docs/guides/example.mdx

```diff
@@ -6,34 +6,34 @@ sidebar_position: 2
+1. [Create a Warp account](get-started.mdx). [Accounts](../how/overview.mdx#warp-accounts) are smart contracts that only you and your Warp jobs have permission to access.
 
-2. [Add any assets](get-started.mdx#create-an-account) (in this case, Luna) to your account. 
+2. [Add any assets](get-started.mdx#create-an-account) (in this case, Luna) to your account.
 
-3. [Create a job](get-started.mdx#create-a-job) with the swap parameters such as price per unit, how much Luna you're willing to spend, etc. For this example, suppose that you want to spend 1 Luna to receive at minimum 9500 ASTRO. You can see the [job message in the example below](#the-job-message). 
+3. [Create a job](get-started.mdx#create-a-job) with the swap parameters such as price per unit, how much Luna you're willing to spend, etc. For this example, suppose that you want to spend 1 Luna to receive at minimum 9500 ASTRO. You can see the [job message in the example below](#the-job-message).
 
-4. Using a [query message](#the-query), Create the [condition](#the-condition), where you specify that for your 1 Luna, you want to receive at least 9500 Astro Luna (~0.00010526 Luna per unit). 
+4. Using a [query message](#the-query), Create the [condition](#the-condition), where you specify that for your 1 Luna, you want to receive at least 9500 Astro Luna (~0.00010526 Luna per unit).
 
-5. Submit the job, which includes your transaction, condition, and a reward. The reward is the amount that you will pay the keeper for executing your transaction on your behalf when the price is right. The reward will likely be equal to the gas cost for executing your job plus a little extra for the keeper’s effort. 
+5. Submit the job, which includes your transaction, condition, and a reward. The reward is the amount that you will pay the keeper for executing your transaction on your behalf when the price is right. The reward will likely be equal to the gas cost for executing your job plus a little extra for the keeper’s effort.
 
 6. Now onto the [keeper](./how/overview#keepers) side of things.
-The keepers scan the transaction pool and see which transactions have conditions that resolve to true.
+   The keepers scan the transaction pool and see which transactions have conditions that resolve to true.
 
-7. Once swapping 1 Luna yields you at least 9500 \$ASTRO, the keeper sees that your job condition is now valid. 
+7. Once swapping 1 Luna yields you at least 9500 \$ASTRO, the keeper sees that your job condition is now valid.
 
-8. At this point, the keeper (which can be anyone, including yourself) executes the transaction, swapping the tokens. Now the Luna is in your Warp account, which you can access and withdraw. 
+8. At this point, the keeper (which can be anyone, including yourself) executes the transaction, swapping the tokens. Now the ASTRO is in your Warp account, which you can access and withdraw.
```

Comment:
I only changed `Luna` to `ASTRO`, rest of change are auto applied due to linter. Feel free to create your own fix using `Astro` as I'm not clear on when all cap.

Reply:
Hello @luuuu-learning!

Yes, you are correct. Thank you for catching this. Merging. 👍